src/NServiceBus.AcceptanceTesting/ScenarioWithContext.cs

```diff
@@ -80,45 +59,72 @@ public Task<IEnumerable<TContext>> Run(TimeSpan? testExecutionTimeout = null)
 
             Console.WriteLine("Total time for testrun: {0}", sw.Elapsed);
 
-            return runDescriptors.Select(r => (TContext)r.ScenarioContext);
+            return runDescriptors.Select(r => (TContext) r.ScenarioContext);
```

Comment:
Ok I'll do the PRs. Rabbit is already there

On Sunday, February 28, 2016, Andreas Öhlund <notifications@github.com>
wrote:

> In src/NServiceBus.AcceptanceTesting/ScenarioWithContext.cs
> <https://github.com/Particular/NServiceBus/pull/3514#discussion_r54354804>
> :
>
> > @@ -80,45 +59,72 @@ public Task<IEnumerable<TContext>> Run(TimeSpan? testExecutionTimeout = null)
> >
> >              Console.WriteLine("Total time for testrun: {0}", sw.Elapsed);
> >
> > -            return runDescriptors.Select(r => (TContext)r.ScenarioContext);
> > +            return runDescriptors.Select(r => (TContext) r.ScenarioContext);
>
> Not at all, [image: :+1:] to merge and update downstreams
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3514/files#r54354804>.
>


-- 
*Don't miss the Async/Await Webinar Series*
*• Learn more <http://go.particular.net/l/27442/2016-02-15/681bjl> →*